bin/install_salt_sources.sh

```diff
@@ -6,9 +6,8 @@ if [ "$FLAVOR" != "devel" ]
         if [[ "$DISTRO" =~ ^rhel ]]
             then
                 bin/rhel_unpack_salt.sh
-        elif [[ "$DISTRO" =~ ^sles ]]
+        elif [[ "$DISTRO" =~ ^(sles|leap) ]]
             then
-                /root/bin/install_salt.sh
```

Comment:
Is it correct to remove this here for the `sles` case?

Reply:
That script was removed 6 years ago (https://github.com/openSUSE/salt-toaster/commit/07ad0ab3cd42e22fb4dc652e8fd9150acf302883)

Reply:
Thanks for clariyfing. There was no mention about this so I wasn't sure here.